src/libraries/System.Net.WebSockets.Client/src/System/Net/WebSockets/ClientWebSocketOptions.cs

```diff
@@ -25,6 +26,31 @@ public sealed class ClientWebSocketOptions
         internal X509CertificateCollection? _clientCertificates;
         internal WebHeaderCollection? _requestHeaders;
         internal List<string>? _requestedSubProtocols;
+        private Version _version = Net.HttpVersion.Version11;
+        private HttpVersionPolicy _versionPolicy = HttpVersionPolicy.RequestVersionOrLower;
+
+        public Version HttpVersion
+        {
+            get => _version;
+            [UnsupportedOSPlatform("browser")]
+            set
+            {
+                ThrowIfReadOnly();
+                ArgumentNullException.ThrowIfNull(value);
+                _version = value;
+            }
+        }
+
+        public HttpVersionPolicy HttpVersionPolicy
+        {
+            get => _versionPolicy;
+            [UnsupportedOSPlatform("browser")]
+            set
+            {
+                ThrowIfReadOnly();
+                _versionPolicy = value;
+            }
+        }
```

Comment:
These properties should be moved down to below the ctor where all the other properties are.